### PR TITLE
Solidjs/counter example

### PR DESCRIPTION
**Pull request by hdJerry**
# Background
We want to launch each kit with an example of a component with state and good tests. We've chosen a counter for this reason.

# Acceptance
- [x] Create a counter component
- [x] The counter should have an increment, decrement, and reset button
- [x] The counter should display its current value
- [x] The counter should be controlled by the global state manager
- [x] The counter should have a storybook story
- [x] The counter should have a set of tests demonstrating how to verify all its features

# Mock
Screen Shot 2022-04-05 at 9 48 58 AM

Screenshot 2022-10-21 at 10 15 56

### storybook
Screenshot 2022-10-21 at 13 34 31

### CODE REVIEW

File: starters/solidjs-tanstack-tailwind/src/components/CounterExample/Counter.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { describe, it, expect } from 'vitest';
+import CounterExample from './index';
+
+describe('', () => {
+  it('should mount', async () => {
+    const wrapper = await render(() => <CounterExample />);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should mount', async () => {
```
Comment (mbicknese) [inline starters/solidjs-tanstack-tailwind/src/components/CounterExample/Counter.spec.jsx:11]:
```suggestion
  it('should increment and display new count', async () => {
```

File: starters/solidjs-tanstack-tailwind/.storybook/preview.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
   template,
   createComponent
 } from "solid-js/web"
-
+import '../src/index';
```
Comment (mbicknese) [inline starters/solidjs-tanstack-tailwind/.storybook/preview.js:9]:
```suggestion
import '../src';
```

**Review by mbicknese — APPROVED** [review]

File: starters/solidjs-tanstack-tailwind/.storybook/preview.js (hunk 2 above)
Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/.storybook/preview.js:9]: sorry, its meant to be index.css.LOL

File: starters/solidjs-tanstack-tailwind/src/components/CounterExample/Counter.spec.jsx (hunk 1 above)
Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/src/components/CounterExample/Counter.spec.jsx]: LOL. I didn't see this

File: starters/solidjs-tanstack-tailwind/src/pages/Home.jsx [hunk 3]
```diff
@@ -2,20 +2,25 @@ import { NavLink } from '@solidjs/router';
...
-      <NavLink
-        href="/api-example"
-        class="hover:text-blue-500 transition-colors delay-100"
-      >
-        See API example component
-      </NavLink>
-    </div>
+    <>
+      <header class="flex justify-center items-center text-white my-5 mx-auto bg-blue-500  w-full lg:w-[75%] p-4 text-lg ">
+        SolidJs, Tanstack and Tailwind Starter kit
```
Comment (vyktoremario) [inline starters/solidjs-tanstack-tailwind/src/pages/Home.jsx:7]: I guess if we decided not to use Solid-query, we should remove `Tanstack` from the header?

Comment (hdJerry) [reply starters/solidjs-tanstack-tailwind/src/pages/Home.jsx:7]: Yes, i was hoping we create ticket to change everything related to it, not just the header. 

What do you think?

Comment (vyktoremario) [reply starters/solidjs-tanstack-tailwind/src/pages/Home.jsx:7]: That sounds good to me

**Review by vyktoremario — APPROVED** [review]

**Pull request merged**